include/openspace/util/updatestructures.h

```diff
@@ -31,60 +31,72 @@
 
 namespace openspace {
 
+class Deferredcaster;
 class VolumeRaycaster;
 
 struct InitializeData {};
 
 struct TransformData {
-    glm::dvec3 translation;
-    glm::dmat3 rotation;
-    double scale;
+	glm::dvec3 translation;
```

Comment:
The editor seems to have accidentally replaced spaces with tabs

Reply:
Fixed.